src/components/backend-ai-storage-proxy-list.ts

```diff
@@ -24,6 +24,7 @@ import {BackendAiStyles} from './backend-ai-general-styles';
 import {IronFlex, IronFlexAlignment} from '../plastics/layout/iron-flex-layout-classes';
 import './backend-ai-dialog';
 import './lablup-progress-bar';
+import { booleanLiteral } from '@babel/types';
```

Comment:
@YejeeHa 
Coud you explain the purpose of this code?

Reply:
I don't know why it was added cause I really didn't add that code line. I think it was added automatically when I clicked the automatic completing word 'booleanLiteral' by mistake. I will delete it. Thank you for letting me know.